includes/export/abstract-wc-csv-batch-exporter.php

```diff
@@ -72,7 +73,7 @@ public function get_file() {
 	public function export() {
 		$this->send_headers();
 		$this->send_content( $this->get_file() );
-		@unlink( $this->get_file_path() );
+		@unlink( $this->get_file_path() ); // phpcs:ignore WordPress.VIP.FileSystemWritesDisallow.file_ops_unlink, Generic.PHP.NoSilencedErrors.Discouraged
```

Comment:
Similar to the above, instead of silencing the error, should we maybe show an error if unable to delete the file? I haven't checked the context in which this code runs to know if this is possible.